app/views/pursuits/index.html.haml

```diff
@@ -0,0 +1,7 @@
+%h1 Pursuits
+
+- @pursuits.each do |pursuit|
+  .pursuit{:id => "#{pursuit.name.downcase.gsub(" ", "_")}"}
```

Comment:
Hash attribute should start with one space after the opening brace<br>Hash attribute should end with one space before the closing brace